src/index.js

```diff
@@ -47,9 +47,20 @@ export default function loader(content, map, meta) {
   plugins.push(icssParser({ urlHandler }));
 
   if (options.import !== false && exportType === 'full') {
+    const resolver = this.getResolve({
+      mainFields: ['css', 'style', 'main', '...'],
+      mainFiles: ['index', '...'],
+      extensions: ['.css'],
+      restrictions: [/\.css$/i],
```

Comment:
This restriction broke `@import` for files with an extension other than `.css` (#1164). Can we remove it (#1165)?